pkg/validation/validation.go

```diff
@@ -11,22 +11,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package contour
+package validation
```

Comment:
Does this need to be public? Why not leave in `internal`?

Reply:
@stevesloka I put the validation funcs in a public pkg due to https://github.com/projectcontour/contour-operator/pull/204#discussion_r578122093. I can move to internal if you prefer.

Reply:
I'm not strongly wedded to having them public, just seems like it would be useful for anyone actually consuming Contour objects to be able to use the standard validation rules rather than having them private.